packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -351,6 +355,13 @@ class ExpansionPanelList extends StatefulWidget {
   /// constructor.)
   final Object initialOpenPanelValue;
 
+  /// Padding is applied to panel header when expanded.
+  /// By default, some space is added to the header vertically above and below
```

Comment:
We typically specify the default values as the last sentence/paragraph. Please specify the exact default value, aka the result of `EdgeInsets.symmetric(vertical: 64.0 - _kPanelHeaderCollapsedHeight)`.

Reply:
Done.